### DISCUSSION TITLE
Welcome to Keptn Discussions

### DISCUSSION THREAD

**Discussion by christian-kreuzberger-dtx** [keptn/keptn#2821] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jetzlstorfer** [comment]
This is great! 👍🏼  Thanks Christian for starting this 😃 

Hi, my name is Jürgen 👋🏼 
I am one of the initials contributors of Keptn and now helping others getting started with Keptn. If you have any questions, you can find me in out [Keptn slack](https://slack.keptn.sh) and on Twitter [@jetzstorfer](https://twitter.com/jetzlstorfer). Looking forward to meeting with you all!

**Comment by johannes-b** [comment]
Hello 😃 

I'm Johannes and also one of the core maintainers of Keptn project from day 1 on. I'm mainly responsible for planning the roadmap and working packages, as well as enhancing the general user experience.

**Comment by christian-kreuzberger-dtx** [comment]
Hi :call_me_hand: 

my name is Christian and I am one of the core maintainers for the Keptn project, though not from day 1 on :wink: 

My primary focus is on making sure that the CI pipeline works, integration tests, but I also write the occassional line of Go code :see_no_evil: 
In addition, I'm very active on GitHub and I'm writing new issues, answering questions, reviewing PRs, etc... :superhero: 

You can find me on the [Keptn Slack](https://slack.keptn.sh/) and of course here on GitHub.

**Comment by oleg-nenashev** [comment]
I pinned this thread in the top of the forum. Happy to contribute! 🚀
